src/interfaces/IOptimismMintableERC20.sol

```diff
@@ -5,6 +5,7 @@ import {IERC165} from
     "../../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/utils/introspection/IERC165.sol";
 
 /// @title IOptimismMintableERC20
+/// @custom:security-contact security@morpho.org
```

Comment:
btw the author is missing here (to me it's useless anyway, but we need to be consistent)